taker-frontend/src/components/History.tsx

```diff
@@ -154,8 +154,12 @@ const CfdDetails = ({ cfd, connectedToMaker, displayCloseButton }: CfdDetailsPro
                 <Table size="sm" variant={"unstyled"}>
                     <Tbody>
                         <Tr>
-                            <Td><Text as={"b"}>Total fees</Text></Td>
-                            <Td textAlign="right"><BitcoinAmount btc={cfd.accumulated_fees} /></Td>
+                            <Td><Text as={"b"}>Estimated fees</Text></Td>
+                            <Td textAlign="right">
+                                <Tooltip label={"includes transaction fees"} placement={"right"}>
```

Comment:
🚧 Not sure why but this tooltip seems to break everything. Removing it makes the side load again.

```
Uncaught Error: React.Children.only expected to receive a single React element child.
    onlyChild React
    Tooltip chakra-ui-tooltip.esm.js:290
    React 12
    unstable_runWithPriority scheduler.development.js:468
    React 5
    listener useLatestEvent.ts:13
    useLatestEvent useLatestEvent.ts:16
    React 5
    unstable_runWithPriority scheduler.development.js:468
    React 4
    unstable_runWithPriority scheduler.development.js:468
    React 6
    <anonymous> main.tsx:9
react.development.js:1139:12
```

If you move the tooltip around `Td` it works.

Reply:
Oh, interesting. On my machine it works, but I generally had some trouble with chakra tooltips causing errors; so far it was not permanent errors though, i.e. the page loaded, but the logs showed an error when hovering over the tooltip. 

I'll move it around the `Td` if that helps.

Reply:
If I wrap it around the `Td` I get this error (that is not fatal though):

```
react-dom.development.js:67 Warning: validateDOMNesting(...): <span> cannot appear as a child of <tr>.
    at span
    at DefaultPortal2 (http://localhost:3000/node_modules/.vite/@chakra-ui_react.js?v=8421fb55:1401:36)
    at Portal (http://localhost:3000/node_modules/.vite/@chakra-ui_react.js?v=8421fb55:1464:28)
    at PresenceChild (http://localhost:3000/node_modules/.vite/chunk-QRD5N5TB.js?v=8421fb55:6477:21)
    at AnimatePresence (http://localhost:3000/node_modules/.vite/chunk-QRD5N5TB.js?v=8421fb55:6556:21)
    at http://localhost:3000/node_modules/.vite/@chakra-ui_react.js?v=8421fb55:20820:17
...
```

It seems Chakra somehow has a `span` for the tooltip which is not allowed as sub-element in html tables.
When navigating through the dom-tree in the browser I was unable to find the span that causes the problem though. 

The chakra documentation states

> If the children of Tooltip is a string, we wrap with in a span with tabIndex set to 0, to ensure it meets the accessibility requirements.

https://chakra-ui.com/docs/overlay/tooltip#usage

But I don't think that is the case here. I think we should report this as bug upstream.